content/pt-br/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -18,122 +15,91 @@ princípios do Kubernetes, notavelmente o [ciclo de controle](/docs/concepts/#ku
+O Controlador normalmente é executado fora do {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}}, assim como você executaria qualquer aplicação containerizada.
 Por exemplo, você pode executar o controlador no seu cluster como um *Deployment*.
 
-## Usando um Operador
+## Usando um operador
 
-Uma vez que você tenha um Operador instalado, usaria-o adicionando, modificando
-ou apagando a espécie de recurso que o Operador usa. Seguindo o exemplo acima,
-você configuraria um *Deployment* para o próprio Operador, e depois:
+Depois de implantar um operador, você o usaria adicionando, modificando ou excluindo o tipo de recurso que o operador usa. Seguindo o exemplo acima,
+você configuraria um *Deployment* para o próprio operador, e depois:
 
 ```shell
-kubectl get SampleDB                   # encontra a base de dados configurada
+kubectl get SampleDB                   # encontrar banco de dados configurados
 
-kubectl edit SampleDB/example-database # mudar manualmente algumas definições
+kubectl edit SampleDB/example-database # alterar manualmente algumas configurações
 ```
 
-&hellip;e é isso! O Operador vai tomar conta de aplicar
-as mudanças assim como manter o serviço existente em boa forma.
+&hellip;e é isso! O Operador cuidará de aplicar as alterações, bem como manter o serviço existente em bom estado.
 
-## Escrevendo o seu próprio Operador {#escrevendo-operador}
+## Escrevendo o seu próprio operador
 
-Se não existir no ecosistema um Operador que implementa
-o comportamento que pretende, pode codificar o seu próprio.
-[Qual é o próximo](#qual-é-o-próximo) você vai encontrar
-alguns *links* para bibliotecas e ferramentas que pode usar
-para escrever o seu próprio Operador *cloud native*.
+Se não houver um opeardor no ecosistema que implemente o comportamento Desejado, você pode programar o seu próprio.
 
-Pode também implementar um Operador (isto é, um Controlador) usando qualquer linguagem / *runtime*
-que pode atuar como um [cliente da API do Kubernetes](/docs/reference/using-api/client-libraries/).
+Você também pode implementar um operador (ou seja, um Controlador) usando qualquer linguagem/*runtime* que possa atuar como um [cliente para a API do Kubernetes](/docs/reference/using-api/client-libraries/).
 
+A seguir estão algumas bibliotecas e ferramentas que você pode usar para escrever seu próprio operador nativo de nuvem.
 
+> **_NOTE:_**  Esta seção contém links para projetos de terceiros que fornecem funcionalidades requeridas pelo Kubernetes. Os autores do projeto Kubernetes não são responsáveis por esses projetos, que estão listados em ordem alfabética. Para adicionar um projeto a esta lista, leia o [guia de conteúdo](/docs/contribute/style/content-guide/#third-party-content) antes de enviar uma alteração. [Mais informações](/docs/concepts/extend-kubernetes/operator/#third-party-content-disclaimer).
 
```

Comment:
In the original file there is "{{% thirdparty-content %}}" header before thirdparty itens.